classes/ImageManager.php

```diff
@@ -231,6 +232,13 @@ public static function resize($src_file, $dst_file, $dst_width = null, $dst_heig
             return !($error = self::ERROR_MEMORY_LIMIT);
         }
 
+        //If no padding one of the two destination size need to be changed (one is already good but we redo it for both to be generic).
+        if ($no_padding)
+        {
```

Comment:
Sorry, I was stuck on old Prestashop recommandations. I had post on the forum a while ago when I discovered spaces instead of tabs in new Prestashop versions whereas documentation was still advocating tabs. Nobody answered.

Should I pull again?